controllers/queryHandler.js

```
@@ -0,0 +1,34 @@
+const {INVALID} = require('./utils');
+const QUERY_ASC = 'asc';
+const QUERY_DESC = 'desc';
+const QUERY_DATE = 'date';
+const QUERY_VIEWS = 'views';
+const QUERY_LIKES = 'likes';
+const QUERY_ALPHABETICAL = 'alphabetical';
+
+exports.handleSort = function(sort, order, query) {
+  if (sort !== undefined) {
+    if (order !== QUERY_ASC && order !== QUERY_DESC) {
+      return INVALID;
```

Throw an error instead of return invalid.

---

Done.